frontend/src/components/Grades/GradesTables.js

```diff
@@ -0,0 +1,94 @@
+import moment from "moment";
+import "moment/locale/pl";
+import React from "react";
+import {
+    getGrades,
+    getUnmarkedActivities,
+    GRADES_TABLE_DATE_FROMAT,
+} from "../../storage/gradesTable";
+import {
+    getActivityTypeName,
+    GradesTableType,
+    percentagesToGrade,
+} from "../../utils/constants";
+import { TableContainer } from "../PointsPage/Table/TableStyle";
+
+const GradesTableContent = (tableType, row, idx) => {
+    let tableContent = {
+        id: idx + 1,
+        name: row.activityName,
+        ...(tableType === GradesTableType.GRADES_TABLE && {
+            grade: percentagesToGrade(row.points / row.maxPoints),
+        }),
+        points:
+            tableType === GradesTableType.GRADES_TABLE
+                ? row.points + "/" + row.maxPoints
+                : row.maxPoints,
+        date: moment(row.date, GRADES_TABLE_DATE_FROMAT).fromNow(),
+        activityType: getActivityTypeName(row.activityType),
+    };
+    return Object.entries(tableContent);
+};
+
+const GradesTableHeaders = (tableType) => {
+    return [
+        "Nr",
+        "Aktywność",
+        ...(tableType === GradesTableType.GRADES_TABLE
+            ? ["Ocena", "Punkty"]
+            : []),
+        ...(tableType === GradesTableType.UNMARKED_ACTIVITIES_TABLE
+            ? ["Punkty do zdobycia"]
+            : []),
```

Comment:
A tutaj jak masz w linijce 39 pustą listę to nie możesz tam wrzucić "Punkty do zdobycia" i wyrzucić linijki 40-42 ?